VultisigApp/VultisigApp/View Models/KeysignViewModel.swift

```diff
@@ -79,7 +79,16 @@ class KeysignViewModel: ObservableObject {
         self.customMessagePayload = customMessagePayload
         self.encryptionKeyHex = encryptionKeyHex
         let isEncryptGCM =  await FeatureFlagService().isFeatureEnabled(feature: .EncryptGCM)
-        self.messagePuller = MessagePuller(encryptionKeyHex: encryptionKeyHex,pubKey: vault.pubKeyECDSA, encryptGCM:isEncryptGCM)
+        
+        // Use the correct public key based on signing type
+        let pubKeyForMessagePuller: String
```

Comment:
why these changes? we technically don't need PubKey here , it can be removed , so don't change it